includes/rest/class-webfinger.php

```diff
@@ -24,6 +24,7 @@ public static function init() {
 	 * Register routes
 	 */
 	public static function register_routes() {
+		\l( 'register webfinger' );
```

Comment:
remove debug function

Reply:
Oh haha good thing it wasn't a spicier one! ;)

Reply:
I'll have to figure out a way to run these tests rather than crossing my fingers!